commands/log.php

```diff
@@ -35,6 +35,8 @@ function terminal_input(String $inputquery, String $rootPath): void
         // Dont run empty inputs
         if (strlen($inputquery ) < 2)
             return;
+		if (substr($inputquery, 0, 3) === "cd ")
+			return;
```

Comment:
Instead of nested if conditions, you can write it like so:

```
 if (strlen($inputquery ) < 2 || substr($inputquery, 0, 3) === "cd "))
            return;
```

Also, I can't immediately see how this fixes the problem without causing new bugs, so I'll have to test this once the refactoring is done. :)